GameData/BoringCrewServices/Parts/Starliner/BCS_Centauri_LAE.cfg

```diff
@@ -45,7 +45,7 @@ PART
 	maxTemp = 2000 // = 3600
 	bulkheadProfiles = size0, srf
 
-	tags =  Starliner Boring ?BCS engine RS88 Abort
+	tags =  Starliner Boring BCS engine RS88 Abort
```

Comment:
as Zorg said [here](https://github.com/zorg2044/BoringCrewServices/pull/28#issuecomment-2408606183)

Reply:
Closed the pull request